packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -832,6 +832,105 @@ void main() {
     expect(find.text('BottomSheet'), findsNothing);
   });
 
+  // Regression test for https://github.com/flutter/flutter/issues/87708
```

Comment:
Nit: Can you add this comment to the other test as well?

Reply:
done